code/game/machinery/computer/cargo.dm

```diff
@@ -235,6 +235,13 @@ For vending packs, see vending_packs.dm*/
 				A.anchored = 1
 				src.transfer_fingerprints_to(A)
 				qdel(src)
+	if(istype(I, /obj/item/toy/lotto_ticket))
+		var/obj/item/toy/lotto_ticket/T = I
+		for(var/mob/V in hearers(src))
+			V.show_message("<b>[src]</b>'s monitor flashes, \"Withdrawing [T.winnings] credits from the Lottery account!\"")
+		dispense_cash(T.winnings, get_turf(src))
```

Comment:
I just like the cargo interaction as if it's a gas station you bought a lotto ticket at. You could call for a security escort if it's a big winning. The intent is to boost player interaction of course

Reply:
I do get that idea, although at a real life gas station the clerk doesn't actually hand over money from the til. They can't skim $200 off the top.

It just seems like an extra layer of risk when you would get the same level of interaction from having to turn in your ticket at cargo but with a direct deposit to your account instead.

Reply:
Like I said, you could use the ticket in-hand to set your account number on it before turning it into the machine. The techie could still change the number but if they did that it would show up in financial records too from the ticket purchase being on one account and the payout being to a different account.

Reply:
I'd say a skimmed jackpot would be preferable to a cargo tech who just put his own account on the ticket